pyproject.toml

```diff
@@ -10,13 +10,17 @@ authors = [
     "Jonathan C. Lau",
     ]
 readme = "README.md"
+packages = [
```

Comment:
I don't _think_ this should be necessary but even if not can keep it if you want